### DISCUSSION TITLE
Python version manager for Termux.

### DISCUSSION THREAD

**Discussion by gphg** [termux/termux-packages#22366] [Q&A] [answered]
I need specific Python 3 version.

Termux using very latest version that aren't likely supported by some module I'm using.

There are many Python version manager, and my problem is quite clear: most of them are not working.

I need recommendations.

**Comment by TomJo2000** [comment, marked as answer]
Version managers do not work on Termux as our packages need to be patched to be able to run under the constraints of an Android app.

Older versions of Python get retired to the [TUR](https://github.com/termux-user-repository/tur) and are available through that channel.
```bash
pkg up
pkg i tur-repo
pkg i python3.11
```
